### ISSUE TITLE
Splitting up in multiple containers?

### ISSUE THREAD

**Issue by wbg** [pimcore/docker-pimcore-demo-standalone#10]
Hi,
Was it an important design decision to keep everything in one (fat) container?
If you are open to splitting it into multiple containers, i would be happy to work on it and provide a pull request as we are thinking about switching to docker for production deployments.
Best,
Roman

**Comment by brusch** [comment]
Hi, 
No this was not a design decision, just out of comfort and for demonstration purposes :) So we're definitely open for a more modularized / containerized architecture :) 

Thanks!

**Comment by DerDummePunkt** [comment]
Jumping in here,
We put some work into an easy configureable docker-compose stack for pimcore, we thought we'd share, so [we forked this repo](https://github.com/Riodigital-de/docker-compose-pimcore-demo)

Take a look and tell me what you think or if you're interested in a pull request.

**Comment by almereyda** [comment]
I will have a go at your fork in our production environment and see how adaptable it is.
Generally I think the README can need a little more love, but in general does the composition follow very good practices.

**Comment by DerDummePunkt** [comment]
Yeah, the readme was mostly intended to highlight some of the more uncommon things we usually do in our docker compose stacks, like having multiple Dockerfiles for different purposes. It's indeed not a fully fledged readme.
Writing good documentation is hard, so it's always helpful to hear another perspective, You're welcome to open up any number of issues in our repo and tell us what you think what information is missing in the readme, I'd be happy to help out.
